vignettes/introduction-to-partition.Rmd

```diff
@@ -69,56 +71,90 @@ df <- simulate_block_data(
   upper_corr = .6,
   n = 100
 )
+```
+
+In a heatmap showing the correlations between the simulated variables, blocks of correlated features are visible:
+
+```{r}
+ggcorrplot::ggcorrplot(corr(df))
+```
+
+Many types of data follow a pattern like this. Closely related to the block correlation structure found in genetic data is that found in microbiome data. The data set `baxter_otu` has microbiome data on 172 healthy patients. Each row represents a patient, and each column represents an Operational Taxonomic Unit (OTU). OTUs are species-like relationships between bacteria determined by analyzing their RNA. Each cell in the data represents the logged-count of the OTU found in the patient's stool sample, and there are 1,234 OTUs in all.
+
+```{r}
+baxter_otu
+```
+
+While not as apparent as simulated data, blocks of correlation also appear in these data; these bacteria tend to group together or apart in the microbiomes of participants. Here are the first 200 OTUs:
 
-df
+```{r}
+correlation_subset <- corr(baxter_otu[, 1:200])
+ggcorrplot::ggcorrplot(correlation_subset, hc.order = TRUE) + ggplot2::theme_void()
 ```
 
-partition takes a data frame and a threshold--the minimum amount of information a reduced variable must explain to be created--and reduces the data to as few variables as possible. The threshold prevents information loss: each reduced variable must still explain at least that much.
```

Comment:
partition takes a data frame and a threshold, the minimum amount of information a reduced variable must explain to be created, and reduces the data into as few variables as possible. The threshold prevents excessive information loss because each reduced variable must still explain at least that much.